apiserver/facades/agent/provisioner/provisioner.go

```diff
@@ -105,8 +81,8 @@ type ProvisionerAPI struct {
 	mu sync.Mutex
 }
 
-// NewProvisionerAPI creates a new server-side ProvisionerAPI facade.
-func NewProvisionerAPI(stdCtx stdcontext.Context, ctx facade.ModelContext) (*ProvisionerAPI, error) {
+// MakeProvisionerAPI creates a new server-side ProvisionerAPI facade.
```

Comment:
It's a new convention that Tom and Harry are pushing. Especially for facade creation methods that can return an error, we're favouring calling them `makeFooAPI` instead of `newFooAPI`.